crates/rolldown/src/utils/parse_to_ecma_ast.rs

```diff
@@ -52,8 +52,8 @@ pub fn parse_to_ecma_ast(
     ModuleType::Ts => (source.try_into_string()?, OxcParseType::Ts),
     ModuleType::Tsx => (source.try_into_string()?, OxcParseType::Tsx),
     ModuleType::Css => {
-      let content = "export {}".to_string();
-      (content, OxcParseType::Js)
+      has_lazy_export = true;
+      ("({})".to_owned(), OxcParseType::Js)
```

Comment:
`{}` will be parsed as `blockStatement` so using this trick generate a `ObjectExpr`, oxc will unwrap it in codegen.